identhandler/modules/visidentifier/visidentifier.go

```diff
@@ -273,8 +272,8 @@ func (instance *Instance) handleConnection(url string) {
 
 		instance.wg.Done()
 
-		select {
-		case err := <-instance.wsClient.ErrorChannel:
+		{
```

Comment:
Remove bracket

Reply:
can't remove in this case we will have compilation error
identhandler/modules/visidentifier/visidentifier.go:260:9: goto reconnect jumps over declaration of err at identhandler/modules/visidentifier/visidentifier.go:275:7